### ISSUE TITLE
QnA prototype

### ISSUE THREAD

**Issue by aaronhans** [cagov/covid19#462]
Create working version of Answers suggested to users when they search based on their query. This should:

- Use design from Art
- Base data on API that uses Azure QnA db Aaron created
- Be integrated into our search results UI without influencing performance negatively

**Comment by aaronhans** [comment]
I like what we have working now, we updated search so it retrieves both basic search results and the Q&A results without another page load: https://as-cdt-pub-acovid-w-p-001-staging.azurewebsites.net/search/?q=testing 

Todo:
- [ ] Get and implement design guidance from @artuoma 
- [ ] Compare results and UI vs Yext
- [ ] Review our content and see what we need to change to improve it

I'll bug the content team after we compare to Yext this afternoon

**Comment by abquirarte** [comment]
From summary:

**Comment by aaronhans** [comment]
We integrated the entire dataset from yext and some suggestions from Art. Here are the remaining known issues:

user testing blockers are:
- [ ] bad links in data, unlinked urls in data, other answer quality issues
- [ ] get Arts approval on initial UI
- [ ] fix slow initial load due to coldstart

known todo items before would consider launch:
- [ ] add test to answer-proxy API
- [ ] add readme to API
- [ ] add GA tracking for 0 answer results
- [ ] add GA tracking for answer opens

**Comment by aaronhans** [comment]
The quick answers are back up in staging @hilaryhb https://as-cdt-pub-acovid-w-p-001-staging.azurewebsites.net/search/?q=test

@abquirarte and @hilaryhb let's identify a set of things we want to change after we have some user feedback and we have time to review it internally.

Thank you @TamaraIDEJA for going over things we should cover in order to be helpful to the call center process. I opened this task to do some data work and present something to help out the call center: https://github.com/cagov/covid19/issues/546

**Comment by abquirarte** [comment]
Any way to filter what comes up first based on key words? For example, it bothers me that this is the first thing that comes up when I type in testing: 
![image](https://user-images.githubusercontent.com/9556188/82471070-0db76880-9a7b-11ea-9b0b-c2b59eef894e.png)

@aaronhans ☝️ 

Super excited to user test this! 

- What do people expect to see in this interaction? 
- Is it clear?
I'm sure @hilaryhb and @artuoma have better ideas lol

**Comment by aaronhans** [comment]
Sure @abquirarte we have full control of this system. 

We could:
- Include the page title prominently as part of the answer so it is obvious this is about schools not virus tests
- Eliminate this answer from our knowledge base
- Keep it in the knowledge base but see if we can make it only come up for education related searches

**Comment by aaronhans** [comment]
We ran this through user testing and got some feedback:

QA did not return results that felt relevant or correct
- 1 person typed in “can i walk my dog” and the QA was exactly that - only 1 case
- ½ people typed in full questions or statements
- People felt like there were doing something wrong when they didn’t get the right results
- People want a link to the page with more info from the QA

Possible fixes:

- Adding a link to the page is straightforward. 
- Bad answer quality can be remedied by adopting a continuous process that reviews any search interaction that "fails". We might define failure as not having any quick answers or presenting some but the user does not click on any. We can record all queries that meet these requirements and see if they reveal content or crawling shortcomings we can remedy.

If we are prepared to invest in this process we can continue and I'll open new tasks to address these shortcomings, work with the content team to integrate reports into their workflow, etc.

If we are not prepared to invest further because the feedback was too negative we can stop this project at this point. We have learned valuable info. We used the manually curated dataset yext created from our data so failing due to data quality represents the scale of that task.

I'll put this prototype task into done for now and we can decide on next steps @abquirarte
